code/src/main/java/frc/robot/commands/DrivetrainControllerCommand.java

```diff
@@ -41,17 +41,11 @@ public void execute() {
 
         _drivetrain.move(deadZones(_controller.getY(Hand.kLeft) * _drivetrain.getMaxSpeed() * .5, 0.2),
```

Comment:
Can the magic numbers here be constants, and maybe make the `getMaxSpeed() * .5` a local variable?

Reply:
I can just reduce the speed throttling because this is for a competition.  Make the deadzone a constant